h5p-bildetema-words-topic-image/src/components/TopicImageSVG/TopicImageSVG.module.scss

```diff
@@ -41,8 +41,12 @@
 
 .imageContainer {
   position: relative;
-  max-height: calc(100vh - 2rem);
+  max-height: calc(100vh - 5rem);
   width: 80%;
+
+  @include breakpoint-min($small) {
+    max-height: calc(100vh - 11rem);
```

Comment:
Are these numbers so large to compensate for the sticky header? Can we add a hint of the intent to the code somehow?

Reply:
We can use a variabel perhaps?

Reply:
Adding some comments for now at least